tests/test_octodns_provider_ns1.py

```diff
@@ -521,6 +521,12 @@ def test_data_for_CNAME(self):
 class TestNs1ProviderDynamic(TestCase):
     zone = Zone('unit.tests.', [])
 
+    _CONTINENT_TO_LIST_OF_COUNTRIES = {
+        'OC': ['FJ', 'NC', 'PG', 'SB', 'VU', 'AU', 'NF', 'NZ', 'FM', 'GU',
+               'KI', 'MH', 'MP', 'NR', 'PW', 'AS', 'CK', 'NU', 'PF', 'PN',
+               'TK', 'TO', 'TV', 'WF', 'WS'],
+    }
```

Comment:
Better to just refer to this as `Ns1Provider. _CONTINENT_TO_LIST_OF_COUNTRIES` rather than duplicate it here.